pdns/lua-record.cc

```diff
@@ -219,6 +231,20 @@ class IsUpOracle
       state->first = false;
     }
   }
+  
+  void setWeight(const CheckDesc& cd, string content){
```

Comment:
how about `  void setWeight(const CheckDesc& cd, int weight){` ? Then the exception handling only needs to live in the other spot.

Reply:
pushed the changes